aptos-move/aptos-gas-schedule/src/gas_schedule/transaction.rs

```diff
@@ -167,12 +167,12 @@ crate::gas_schedule::macros::define_gas_parameters!(
         [
             max_execution_gas: InternalGas,
             { 7.. => "max_execution_gas" },
-            20_000_000_000,
+            5_000_000_000,
```

Comment:
You need to bump the gas swchedule version as well I think?

Reply:
shouldn't update the gas feature version, otherwise we create more confusion and potentially enable gated features accidentally.